### PR TITLE
Adding BluuNext font

### PR DESCRIPTION
**Pull request by koundinyad**

### CODE REVIEW

**Comment by ayuhito** [comment]
I think the ID bluu-next would be more appropriate. Otherwise, this PR seems fine!

File: scripts/generic/config.ts [hunk 1]
```diff
@@ -9,14 +9,14 @@
 // type is fontsource-specific e.g. google, league, other. It should be left as "other" unless otherwise specified.
 
 const config = {
-  fontId: "",
-  fontName: "",
+  fontId: "bluu-next",
```
Comment (ayuhito) [inline scripts/generic/config.ts]: Just need to clear up the config file and we can merge this 👍

**Review by ayuhito — CHANGES_REQUESTED** [review]

File: scripts/generic/config.ts (hunk 1 above)
Comment (koundinyad) [reply scripts/generic/config.ts]: Sorry about that. Committed new changes.

**Review by ayuhito — APPROVED** [review]

**Comment by ayuhito** [comment]
Currently, publishing fonts are paused since Google pushed an update that broke some things. But once that's resolved this should be automatically published 👍

**Pull request merged**